.github/workflows/build.yaml

```diff
@@ -12,14 +12,14 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  build-appimage:
+  build-appimage-and-flatpak:
```

Comment:
Please don't rename the job yet, I'll do that afterwards once this is merged. This rename causes a CI failure due to the fact that `release` depends on `build-appimage` and not `build-appimage-and-flatpak`.

Another reason to hate YAML. Why, GitHub, why.